src/components/Toolbar/Toolbar.js

```diff
@@ -236,7 +236,9 @@ export default class Toolbar extends Component {
       },
       renderAddons = [],
     } = this.props;
-
+    const renderSupport = this.props.support.length > 0;
+    const renderSettings = this.props.settings.length > 0;
+    const height = renderSupport && renderSettings ? '100%' : '50px';
     const classes = classnames(namespace, className);
```

Comment:
Instead of an inline style, could the style be applied through a conditional class in https://github.com/carbon-design-system/ibm-security/blob/dev/src/components/Toolbar/_mixins.scss via [classnames](https://www.npmjs.com/package/classnames#usage)? 

Instead of explicitly defining a height of `50px`, perhaps the (`bottom`)[https://github.com/carbon-design-system/ibm-security/blob/dev/src/components/Toolbar/_mixins.scss#L135] can be toggled to achieve the same effect.